### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by Naesstrom** [danielwelch/hassio-gdrive-sync#1]
When I follow the instructions and add `https://github.com/danielwelch/hassio-addons` to hassio all that comes up is the dropbox sync and not the gdrive sync. Not that weird since in that repo the gdrive sync is missing :D

**Comment by danielwelch** [comment]
You’re right, not surprising at all! This repo is in fact totally unmaintained. With that said, if you’d like to test it out locally and contribute, that’d be great.

**Comment by Naesstrom** [comment]
Ah that explains it 😃 
I'll see if I get around to it this weekend!
